web/source/osk/bannerManager.ts

```diff
@@ -112,7 +118,7 @@ namespace com.keyman.osk {
     private selectBanner(state?: text.prediction.ModelChangeEnum) {
       let keyman = com.keyman.singleton;
 
-      if(keyman.modelManager.activeModel) {
+      if(keyman.modelManager.enabled) {
```

Comment:
Please revert this line.

```suggestion
      // Only display a SuggestionBanner when the current language has an active predictive model.
      // ModelManager will never have an active model when predictions are disabled.
      if(keyman.modelManager.activeModel) {
```

There's a semantic difference here:
* `.activeModel` only returns something 'truthy' when there is actually a model for the active language.
* `.enabled == true` doesn't care whether or not there actually **is** a model for the active language, making the `SuggestionBanner` display at inappropriate times.

Obviously, this has been convenient for testing the PR until now, but it's logically inaccurate for a final commit-and-merge.